engine/src/main/java/org/terasology/engine/module/SandboxManager.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2018 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.engine.module;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.terasology.engine.paths.PathManager;
+import org.terasology.module.sandbox.API;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+@API
+public class SandboxManager {
+
+    private static final Logger logger = LoggerFactory.getLogger(SandboxManager.class);
+
+    private PathManager pathManager;
```

Comment:
Maybe you've tried this and/or this class might not be in a place where `@In` would work, but that'd be a nice easy way to initialize this var. Maybe something to try :-)

Reply:
@Cervator I tried this but I got a `NullPointerException`. Is it added to the `CoreRegistry`? At least I didn't find any occurrences of something like `@In private PathManager pathManager`.